status.go

```diff
@@ -174,11 +175,11 @@ func makeLocalConnectionStatusSlice(cm *connectionMaker) []LocalConnectionStatus
 					info = fmt.Sprintf("%-11v %v", "unencrypted", info)
 				}
 			}
-			slice = append(slice, LocalConnectionStatus{conn.remoteTCPAddress(), conn.isOutbound(), state, info})
+			slice = append(slice, LocalConnectionStatus{conn.remoteTCPAddress(), conn.isOutbound(), state, info, lc.OverlayConn.DisplayData()})
```

Comment:
Not sure about `conn`, as `makeLocalConnectionStatusSlice` is executed by the actor process. (can `conn` be modified outside the actor loop?)

The easy fix is to merge `DisplayName` with `DisplayData` into `GetDisplayNameAndAttrs` (as you suggested in the beginning).